.github/stale.yml

```diff
@@ -0,0 +1,60 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are checked if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - security
+  - backlog
+  - "Rule: New"
```

Comment:
why can't a new rule become stale? we can use backlog for those too right?

Reply:
yea, it should be dictated by whatever flow we see ourselves using. So we can be more explicit with the use of `backlog` anywhere applicable, or we can have some implicit ones. Pros and cons of both, so long as we are consistent.

Thoughts

Reply:
```suggestion
```

we will just use backlog as necessary